superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -183,15 +183,15 @@ def _create_model(self, precision):
             return False
 
         try:
+            self._model = BertBenchmarkModel(self._config, self._args.num_classes)
             if enable_fp8:
                 self._fp8_recipe = DelayedScaling(
                     fp8_format=Format[precision.name.strip('FP8_')],
                     amax_history_len=16,
                     amax_compute_algo='max',
                 )
-                self._model = TeBertBenchmarkModel(self._config, self._args.num_classes).to(dtype=torch.float16)
+                self._to_te_model(self._model.to(dtype=torch.float16))
```

Comment:
Can we remove whole TeBertBenchmarkModel class?

Reply:
yes, removed